prework/windows/1_terminal.md

```diff
@@ -19,17 +19,21 @@ CAVEAT: Until Windows adopts more Unix standards, please be aware that any Unix-
 
 ### Install Git for Windows
 
-[Git for Windows](http://msysgit.github.io/) is a lightweight, native set of tools that bring the full feature set of Git to Windows. It includes **Bash**, a popular Unix shell, which you'll use to run the `git` command line just like other Unix environments.
+[Git for Windows](http://msysgit.github.io/) brings the full feature set of Git to Windows. (More on Git in a moment.) It also includes **Bash**, a popular Unix shell. Besides giving you the ability to talk to your computer by typing commands, it also lets you use Git in the command line, just like other Unix environments.
 
-To get started, download and install [Git for Windows](http://msysgit.github.io/). There are a bunch of installation options, but you can use the defaults if you're feeling overwhelmed.
+What's Git? It's a very popular **version control** system. Version control systems let programmers share and collaborate on code. With Git, multiple programmers can work on the same files, and Git keeps track of who made what changes, when. Git is usually used with a website, GitHub, that stores code (in "repositories") and lets programmers review and discuss changes before they are added. In short, Git makes everyone's lives easier.
 
-Once installed, use the Start Menu to launch Git Bash.
+To get started, download [Git for Windows](http://msysgit.github.io/). Once you click the "download" button, you'll see a list of download options.
 
-![](https://i.imgur.com/9CKUGs5.png)
+Look closely at the first two filenames in the list. One says "32 bit", and the other says "64 bit". Those are the different versions for 32-bit and 64-bit Windows. Download the one for your system. If you don't know whether your version of Windows is 32- or 64-bit, [find out](http://windows.microsoft.com/en-us/windows/32-bit-and-64-bit-windows).
+
+Once you get it downloaded, double-click the file to open the installer. It will guide you through several options; the defaults are just fine, so keep on clicking "Next" until it's done.
 
-**TIP:** Don't be distracted by the **Git GUI** program.
+When it's done installing, use the Start Menu to launch Git Bash. **TIP:** search for and open "Git Bash", not "Git CMD" or "Git GUI". Then, pin it to your Start Menu so it's easy to find.
```

Comment:
Git CMD actually didn't show up in my Start Menu, but we can leave that in there, just in case.